### ISSUE TITLE
Debug output looks strange

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest#6701]
Sorry to bring up the debug menu for the 1,000th time, but I simply can't stand the new debug output in c0dd4ea4. I find it irritating.

* No colons, it just word after word; making it hard to parse for no reason. It's also poor English style
* Everything Is Capitalized For No Apparent Reason. The debug output is neither a name nor a title, so why is it capitalized?
* You wrote `Param2` which is *really* annoying (Lua is case-sensitive)

Suggestion:

```
Minetest <VERSION>, FPS: <X>, draw time: <X>ms, draw time jitter: <X>%, view range: <X>, RTT: <X>s
Pos: (<X>, <Y>, <Z>), yaw: <X>° (<CARDINAL DIR> <CARTESIAN DIR>), seed: <X>, pointed: <X>, param2=<X>

```

This is a very lightweight suggestion, I just added some punctation and reverted the annoying capitalization and changed “Dtime jitter”.

To avoid wasting too much time, just give a quick “yes” or “no”.

**Comment by ThomasMonroe314** [comment]
Looks better to me than what was already there. I give a thumbs up.

**Comment by lisacvuk** [comment]
Agreed, current debug is just hard to read.
The one before was better.

**Comment by SmallJoker** [comment]
So when will we revert the entire debug output changes to 0.4.10-like formatting? /s

**Comment by Fixer-007** [comment]
Well, i liked moving everything into two lines, but not other parts, like formatting, upper case, etc.

**Comment by paramat** [comment]
Removing clutter was the whole idea, and allows us to return to 2 lines. The former excessive use of symbols made it messy. It's cleaner and still readable, give it some time to get used to it. Everyone will have a different opinion of how it should be formatted.

> Everything Is Capitalized For No Apparent Reason.

Every word is not capatilised. The first letter is capatilised to give the eye a point to be drawn to, that's why Param2 has a capital. Choosing a capital or not depending on whether that word or phrase appears in code as a variable will result in inconsistency as in your suggestion. 'Pos' is in code as 'pos', and 'pointed' does not occur in code as 'pointed'.

If it helps we can have all-lowercase or all-uppercase, and i'm happy to do that if wanted.
:-1:

**Comment by paramat** [comment]
I'm ok with #6702  if only colons are added.

**Comment by dsohler** [comment]
> and allows us to return to 2 lines. 

Why exactly only two lines? Why not a nigcely formatted table of the things that are crammed into two lines? Those are debugging informations. They should not be shown all the time so a better readable format that overlays some parts of the game content would not be a big deal.

**Comment by paramat** [comment]
Ok i agree with all lower case, except FPS, RTT.
Willtry adding colons too.

**Comment by paramat** [comment]
#6702 merged.
